src/store/index.js

```diff
@@ -0,0 +1,53 @@
+import { defineStore } from 'pinia'
+import { useRoute } from 'vue-router'
+
+const apiUrl = 'http://localhost:5000/tasks'
```

Comment:
yeah 👍
the `vue-cli` project uses "proxying"
```js
const { defineConfig } = require('@vue/cli-service')
module.exports = defineConfig({
  transpileDependencies: true,
  outputDir: 'docs',
  publicPath: '/ttracker/',
  devServer: {
    proxy: {
      '^/api': {
        target: 'http://localhost:5000',
        changeOrigin: true,
        logLevel: 'debug',
        pathRewrite: { '^/api': '/' },
      },
    },
  },
```

I wonder is it possible to have something similar with the `vite-cli` version?

The `const apiUrl = '/api/tasks'` value is universal and works in "production" (provided that we have the backend at the `/api/tasks`, and the `localhost...` version, won't (unless the backend is on the same server at the 5000 port 😄 of course)

This is a "nice to have", if it is an easy feature.
Otherwise just ignore this 👍